source/codegen/metadata/nixnet/custom_proto.mako

```diff
@@ -0,0 +1,31 @@
+message u32Array {
```

Comment:
Messages need to be `PascalCased`: `U32Array`, `RefArray`.

Reply:
Done.